scripts/build-addon.js

```diff
@@ -30,7 +31,16 @@ let inputOptions = {
         { src: "src/*.js", dest: OUTDIR },
         { src: "src/icons/*", dest: `${OUTDIR}/icons` },
       ]
-    })
+    }),
+    {
+        name: "watch-external",
```

Comment:
Previously rollup watch would only reload the extension if the files used in the rollup build were changed (KeywordTree*.js) this makes sure it reloads if background.js / api.js etc change